src/assets/scss/components/_videoComponent.scss

```diff
@@ -39,7 +40,7 @@
 /* Background Header */
 .img-bg-header {
 	background: linear-gradient(rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)), url("../../../public/images/bg-header.jpg") center center;
```

Comment:
This is not the way to make it work, that path does not compute once it is compiled. You need to instruct sass to disable url handling. Could you remove the `file-loader` and the `image-webpack-loader` dependencies and their webpack tasks.
To fix the css-loader, just set the url flag to false:
```
test: /\.scss$/,
        loaders: ['style-loader', 'css-loader?url=false', 'sass-loader']
```